### DISCUSSION TITLE
Support for MongoDB

### DISCUSSION THREAD

**Discussion by ericschmar** [thanhlmm/refi-app#11] [Ideas]
As far as document datastores go, MongoDB is a massive chunk of use cases. I would love to see something this developer focused for MongoDB instead of the existing crusty applications out there. 

What are your thoughts of adding this functionality? Or perhaps you know of an existing tool that is very similar to this but compatible with Mongo?

**Comment by thanhlmm** [comment]
My vision is RefiApp doing really well at Firestore then we can expand to other NO-SQL databases. I'm really like TablePlus so want Refi App to be something like that but for No-SQL databases.

BTW, which tool do you use with MongoDB, and what is the thing that makes you really anoying?

Thank you for your suggestion! 🙏

**Comment by ericschmar** [comment]
I've actually used Table Plus for MongoDB. It just doesn't work very well with Cosmos DB (my primary cloud DB provider). I think it needs better options for paging or something--I'm really not sure. 

Besides that, I use Mongo Compass, but I don't really like because it's slow, and the UI is not ideal. I mean it's functional, but nothing special.

Refi being developer focused is what I'm primarily interested in. The things like viewing the object JSON in a legit text editor is huge. Not to mention the command bar. 

So to sum up--it's less so actual problems with my existing programs, and more frustration that they aren't innovating whatsoever.

**Reply by thanhlmm** [reply]
Yeah, I have used MongoDB for a long time ago, and Mongo Compass is quite better than any others tool I know for MongoDb. But as said, it is slow and I think that product is not much focused on the developer.

 I don't work much with MongoDB and FireStore right now so I'm not sure that making Refi work with MongoDB is worth doing, but If many of ppl come to this discussion and think it's useful. I definitely will do it!
 
 Thank you for your insight 🙏
